### ISSUE TITLE
C-Extension-Alternatives broken on 1.9

### ISSUE THREAD

**Issue by jkutner** [jruby/jruby-lint#5]
If you run jruby-lint with JRUBY_OPTS=--1.9 it fails to load the C-Extension-Alternatives page from the wiki.  The underlying error is below.  It gets trapped and discarded. As a result jrlint doesn't detect gems that are problematic.

```
#<NoMethodError: undefined method `read_nonblock' for #<OpenSSL::SSL::SSLSocket:0x7099e017>>
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/protocol.rb:135:in `rbuf_fill'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/protocol.rb:116:in `readuntil'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/protocol.rb:126:in `readline'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/http.rb:2211:in `read_status_line'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/http.rb:2200:in `read_new'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/http.rb:1183:in `transport_request'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/http.rb:1169:in `request'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/http.rb:880:in `get'
/.rvm/gems/jruby-1.6.4/gems/jruby-lint-0.3.1/lib/jruby/lint/libraries.rb:51:in `read_from_wiki'
/.rvm/rubies/jruby-1.6.4/lib/ruby/1.9/net/http.rb:627:in `start'
/.rvm/gems/jruby-1.6.4/gems/jruby-lint-0.3.1/lib/jruby/lint/libraries.rb:50:in `read_from_wiki'
```

**Comment by enebo** [comment]
Ancient report on unsupported JRuby. Clearing this out.
